Tic-Tac-Toe.py

```diff
@@ -0,0 +1,171 @@
+#Tic Tac Toe
+
+#create a 3x3 list of blank spaces
+def new_board():
+    board = [[" ", " ", " "], [" ", " ", " "], [" ", " ", " "]]
+    return board
+
+#print the current board to the user
+def print_board(board):
+    print("", board[0][0], "|", board[0][1], "|", board[0][2])
```

Comment:
>     * is the first empty string required?
Print (by default) inserts a space between item printed in a print statement.   This can be changed by the optional `sep` keyword.  For example `print(item1, item2, sep="")` would eliminate the space and `print(item1, item2, sep='\n')` would insert a new line (\n) between each item.  The empty string used here ensures an initial space is printed to center the first columns value within the cell.

>     * consider using [str.join](https://docs.python.org/3/library/stdtypes.html#str.join) here: `"|".join(board[0])`
+1
 
>     * also consider iterating over the rows in the board instead of hard-coding each one:
> ```
> for row in board:
>     print("|".join(row))
>     print("spacer")
> ```
Iterating like this would print an excess spacer on the last line.  Instead you could use `str.join` again on each row to produce a board. E.g.:
```python
col_sep = '|'
row_sep = "---+---+---\n"
rows = []
for row in board:
    rows.append(col_sep.join(row))
print(row_sep.join(rows))
```
However since the board is a fixed 3x3 layout, I would probably prefer to simply use a multi-line string and fill in the board with string formatting and would also give you a simple way to include column and row headers:
```python
output = """
   0   1   2
A  {0} | {1} | {2}
  ---+---+---
B  {3} | {4} | {5}
  ---+---+---
C  {6} | {7} | {8}
"""
```

Reply:
Yeah, good catch on the extra spacer. Leif caught my mistake as he went to make the changes. I ended up on a mess of map, joins and lambdas, but find your suggestion much easier to read.